api/start.js

```diff
@@ -0,0 +1,4 @@
+const app = require('./index')
+const PORT = 9000
+
+app.listen(PORT)
```

Comment:
Change root project README.md to say use nodemon start.js etc instead of index.js